examples/graphql-interceptors/graphql_interceptors.md

```diff
@@ -0,0 +1,18 @@
+# Interceptors
+
+The GraphQL `interceptors` can be used to execute custom logic before and after the resolver function gets invoked. It can be defined as a read-only service class, which infers the `graphql:Interceptor` service object. The interceptor service class should have the implementation of the `execute(graphql:Context context, graphql:Field 'field)`remote method which is provided by the interceptor service object. The custom logic can be included in this remote method. The interceptors should be provided using the `graphql:ServiceConfig` parameter named `interceptors` which accepts an array of interceptor instances.
```

Comment:
```suggestion
The GraphQL `interceptors` can be used to execute custom logic before and after the resolver function gets invoked. It can be defined as a read-only service class, which infers the `graphql:Interceptor` service object. The interceptor service class should have the implementation of the `execute(graphql:Context context, graphql:Field 'field)`remote method, which is provided by the interceptor service object. The custom logic can be included in this remote method. The interceptors should be provided using the `graphql:ServiceConfig` parameter named `interceptors`, which accepts an array of interceptor instances.
```